### DISCUSSION TITLE
外置播放器的文字为什么会整体发亮呢

### DISCUSSION THREAD

**Discussion by d2033801** [amll-dev/applemusic-like-lyrics#212] [General]
这是内置播放器的情况↓
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/70478207/367a84a1-11c4-458b-854a-2c1490663d32)

这是外置播放器的情况↓
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/70478207/7ce6f9e4-7b35-493b-a003-409db24b227d)

屏幕分辨率为1920*1080，无缩放。使用了伪流体背景
看起来有种hdr色彩被不正确转换导致过曝的感觉
又或者是字体渲染没有抗锯齿导致的？

**Comment by Steve-xmh** [comment]
Apple Music 原版也是使用相同的字体颜色混合方式，不信可以看看 iPad 版本或者 Android 版本

**Reply by d2033801** [reply]
我晓得混色方式没有问题，内置播放模式和自己拿手机的时候也都没有问题。
这个小图看不太清，您可以放大打开我发的图片，在使用外置播放器的时候会感觉文字过曝而直接在网易云用的时候并不会这样

**Reply by Steve-xmh** [reply]
网易云里使用的是默认的混色方式，因为网易云所使用的CEF(Chrome)版本不足以使用 增加变亮（`plus-lighter`）的混色方式，因此回滚了默认的颜色混合方式。其实外置播放器的效果才是正确的，其边缘感到过曝而产生一种锯齿感也是正常行为，理论上如果像素密度够高，这个是看不出来的。

这个是我在 Android 版本的 Apple Music 上截图的效果：
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/39523898/beb3da9f-c629-4b43-a05e-7addaf359846)
这个是我在 Web 版本的 Apple Music 上截图的效果：
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/39523898/2cb76521-05ec-4241-ab07-5ed982d87bc8)

**Reply by d2033801** [reply]
原来如此，是我理解错误了
<del>暗示显示器该加钱了</del>
是否可以通过抗锯齿等方法来减轻影响呢？在低分辨率显示器(24寸1080p)上目前只能通过降低亮度来缓解

**Reply by Steve-xmh** [reply]
其实可以轻微降低一下高亮部分的透明度的，看我心情或者有没有人给个PR了
